examples/http-disable-chunking/http_disable_chunking.description

```diff
@@ -1,2 +1,2 @@
-//This sample demonstrates how to change the Ballerina HTTP client connector's chunking behavior. By default, the HTTP client sends content-length messages. 
+//This sample demonstrates how to change the Ballerina HTTP client endpoint's chunking behavior. By default, the HTTP client sends content-length messages.
```

Comment:
I think we need to rephrase the following: "By default, the HTTP client sends content-length messages."

What's a content-length message? I think the intention here is to say that by default, the HTTP client sends requests with the whole payload, with the content-length set.

Reply:
@chamil321 @pamod Any suggestions on the above or shall we fix these descriptions in a seperate PR?

Reply:
Considering there's a lot of improvements to do for these descriptions, +1 for addressing them in a separate PR. Created #13073 to track this.